src/app/shared/shared.module.ts

```diff
@@ -1,9 +1,20 @@
 import { NgModule } from '@angular/core';
 
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+
 import { BreadcrumbsComponent } from './components/breadcrumbs/breadcrumbs.component';
+import { TimeDurationPipe } from './pipes/time-duration.pipe';
 
 @NgModule({
-  declarations: [ BreadcrumbsComponent ],
-  exports: [ BreadcrumbsComponent ],
+  imports: [ FontAwesomeModule ],
+  declarations: [
+    BreadcrumbsComponent,
+    TimeDurationPipe,
```

Comment:
Perhaps, just `DurationPipe`?

Reply:
Yes, I will move BreadcrumbsComponent to CoreModule